detox/.detoxrc.json

```diff
@@ -0,0 +1,42 @@
+{
+  "testRunner": "jest",
+  "runnerConfig": "e2e/config.json",
+  "configurations": {
+    "ios.sim.debug": {
+      "binaryPath": "../ios/Build/Products/Debug-iphonesimulator/Mattermost.app",
+      "type": "ios.simulator",
+      "device": {
+        "type": "iPhone 11"
+      }
+    },
+    "ios.sim.release": {
+      "type": "ios.simulator",
+      "binaryPath": "../ios/Build/Products/Release-iphonesimulator/Mattermost.app",
+      "build": "cd ../fastlane && NODE_ENV=production bundle exec fastlane ios simulator && cd ../detox",
+      "device": {
+        "type": "iPhone 11"
+      }
+    },
+    "android.emu.debug": {
+      "type": "android.emulator",
+      "binaryPath": "../android/app/build/outputs/apk/debug/app-debug.apk",
+      "build": "cd ../android && ./gradlew assembleDebug assembleAndroidTest -DtestBuildType=debug && cd ../detox",
+      "device": {
+        "avdName": "detox_emu_api_30"
+      }
+    },
+    "android.emu.release": {
+      "type": "android.emulator",
+      "binaryPath": "../android/app/build/outputs/apk/release/app-release.apk",
+      "build": "cd ../android && ./gradlew assembleRelease assembleAndroidTest -DtestBuildType=release && cd ../detox",
+      "device": {
+        "avdName": "detox_emu_api_30"
```

Comment:
we need to document this. I mean that there is a need for an emulator with this name. iOS we could include it in docs but it is pretty standard

Reply:
Yes. It will be included in the docs which is to follow.